arm-sql/2014-04-01/swagger/dataMasking.json

```diff
@@ -0,0 +1,502 @@
+                        "$ref": "#/definitions/DataMaskingRule"
+                    },
+                    "description": "The list of database data masking rules."
+                }
+            },
+            "description": "The response to a list data masking rules request."
+        },
+        "Resource": {
+            "description": "ARM resource.",
+            "properties": {
+                "id": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "Resource ID."
+                },
+                "name": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "Resource name."
+                },
+                "type": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "Resource type."
+                }
+            },
+            "x-ms-azure-resource": true
+        },
+        "ProxyResource": {
+            "description": "ARM proxy resource.",
+            "allOf": [
+                {
+                    "$ref": "#/definitions/Resource"
+                }
+            ]
+        }
+    },
+    "parameters": {
+        "SubscriptionIdParameter": {
+            "name": "subscriptionId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The subscription ID that identifies an Azure subscription."
+        },
+        "ApiVersionParameter": {
+            "name": "api-version",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "description": "The API version to use for the request."
+        },
+        "ResourceGroupParameter": {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group that contains the resource. You can obtain this value from the Azure Resource Manager API or the portal.",
+            "x-ms-parameter-location": "method"
+        },
+        "ServerNameParameter": {
+            "name": "serverName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the server.",
+            "x-ms-parameter-location": "method"
+        },
+        "DataMaskingPolicyNameParameter": {
+            "name": "dataMaskingPolicyName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "enum": [
+                "Default"
+            ],
+            "x-ms-enum": {
+                "modelAsString": false,
+                "name": "DataMaskingPolicyName"
+            },
+            "description": "The name of the database for which the data masking rule applies.",
+            "x-ms-parameter-location": "method"
+        }
+    },
+    "securityDefinitions": {
```

Comment:
this is not related to Data masking. isn't is can be retrieved from some share location?